source/Government.cpp

```diff
@@ -152,6 +159,26 @@ void Government::Load(const DataNode &node)
 
 
 
+// Load a set of reputation titles and return them as a pair of arrays.
+vector<pair<double, string>> Government::LoadReputationTitles(const DataNode &node)
+{
+	vector<pair<double, string>> result;
+	if(!node.HasChildren())
+		return result;
+	for(const DataNode &child : node)
+	{
+		if(child.Size() != 2)
+		{
+			child.PrintTrace("Skipping incorrectly formatted reputation title node.");
+			continue;
+		}
+		result.emplace_back(child.Token(0) == "-infinity" ? numeric_limits<double>::min() : child.Value(0), child.Token(1));
```

Comment:
```suggestion
		result.emplace_back((child.Token(0) == "-infinity" ? numeric_limits<double>::min() : child.Value(0)), child.Token(1));
```
I'm not sure if its actually required for it to be read properly by a computer, but I think it would help making it readable for everyone